src/com/sap/piper/ConfigurationHelper.groovy

```diff
@@ -1,22 +1,25 @@
 package com.sap.piper
 
 class ConfigurationHelper implements Serializable {
-    static ConfigurationHelper loadStepDefaults(Script step){
-        return new ConfigurationHelper(step)
-            .initDefaults(step)
-            .loadDefaults()
+
+    ConfigurationHelper loadStepDefaults() {
+            initDefaults()
+            loadDefaults()
     }
 
-    private Map config = [:]
+    private Map config
+    private Script script
```

Comment:
it was by intension that we don't store the `step`/`script`, this leads to issues

Reply:
iI curious: what issues did you encounter?

Reply:
NonSerializeableExceptions